lib/github.com/diku-dk/linalg/linalg.fut

```diff
@@ -7,6 +7,28 @@
 local module type linalg = {
   -- | The scalar type.
   type t
+
+  ------ CREATING MATRICES/VECTORS ------
```

Comment:
You need a blank like after this or the documentation comment for `veczeros` will not be picked up.

Reply:
Is this fixed now or did I misunderstand?